src/kernels/transformedkernel.jl

```diff
@@ -9,6 +9,12 @@ struct TransformedKernel{Tk<:Kernel,Tr<:Transform} <: Kernel
     transform::Tr
 end
 
+function (k::TransformedKernel)(x, y)
+    x′ = vec(apply(k.transform, reshape(x, :, 1); obsdim=2))
+    y′ = vec(apply(k.transform, reshape(y, :, 1); obsdim=2))
```

Comment:
```suggestion
    x′ = apply(k.transform, x)
    y′ = apply(k.transform, y)
```
All transforms are made to be applied on vectors as well